<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgROSAPIUsers</class>
 <widget class="QDialog" name="DlgROSAPIUsers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Usuarios y grupos ROS API</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Grupos de usuarios</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>602</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="addGroupButton">
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="delGroupButton">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QROSAPIUsersGroupTableWidget" name="groupsTable"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Usuarios</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="2">
       <widget class="QToolButton" name="delUserButton">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QROSAPIUserTableWidget" name="usersTable"/>
      </item>
      <item row="0" column="0">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>428</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="addUserButton">
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;OjO:&lt;/span&gt;&lt;br/&gt;- Los cambios que se hagan aquí, se harán directamente sobre todos los routers.&lt;br/&gt;- El nombre de los usuarios no se pueden cambiar; una vez se ha añadido el usuario (colocando el grupo correctamente) no podrá cambiarse.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QROSAPIUsersGroupTableWidget</class>
   <extends>QTableWidget</extends>
   <header>Widgets/QROSAPIUsersGroupTableWidget.h</header>
  </customwidget>
  <customwidget>
   <class>QROSAPIUserTableWidget</class>
   <extends>QTableWidget</extends>
   <header>Widgets/QROSAPIUserTableWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
